### PR TITLE
Put 4K and 7K MWCs directly under the Tournaments/MWC folder

### LINKED ISSUE #1663: Put 4K and 7K MWCs directly under the Tournaments/MWC folder
current:
```
MWC
  - 20XX
      - 4K
      - 7K
```
proposed:
```
MWC
  - 20XX_4K
  - 20XX_7K
```
I think having the common parent of 20XX makes no sense because the two tournaments aren't any more related than other world cups. if we're filing articles under folders that represent their topic, a folder called "2017" doesn't say a whole lot

also, the index pages for MWC/20XX are missing. this change would solve the problem of having to add a meaningless index page

### PR DESCRIPTION
**Pull request by cl8n**
resolves #1663 

---

### CODE REVIEW

**Comment by Joehuu** [comment]
The images in the news are broken since it got moved to the `img` folder.

**Comment by cl8n** [comment]
doesn't seem very relevant to this PR but I'll fix it with conflict solve

**Comment by cl8n** [comment]
@Joehuu assuming that's what you were referring to? @ 0abbc7f24988bbb36ea3107b6460b6cc3603d2ad

File: news/2016-06-28-osumania-4k-world-cup-2016-registrations-now.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ permalink: http://osunews.tumblr.com/post/146609981558/osumania-4k-world-cup-201
 date: 2016-06-28 16:48:48 +0000
 tumblr_url: http://osunews.tumblr.com/post/146609981558/osumania-4k-world-cup-2016-registrations-now
 ---
-We'd like to welcome all of you to this year's official osu!mania 4K World Cup. As usual, the rules have been adjusted from the previous MWC 4K, so please **[read them carefully](https://osu.ppy.sh/help/wiki/Tournaments/MWC/2016/4K#Ruleset)** before registering for this tournament.
+We'd like to welcome all of you to this year's official osu!mania 4K World Cup. As usual, the rules have been adjusted from the previous MWC 4K, so please **[read them carefully](https://osu.ppy.sh/help/wiki/Tournaments/MWC/2016_4K#Ruleset)** before registering for this tournament.
```
Comment (Joehuu) [inline news/2016-06-28-osumania-4k-world-cup-2016-registrations-now.md]: Nitpick, jump to doesn't work with capital letters.
`#Ruleset` -> `#ruleset`

Comment (cl8n) [reply news/2016-06-28-osumania-4k-world-cup-2016-registrations-now.md]: why am i fixing problems with things im not changing Zz

**Pull request merged**